components/channel_layout/app_route/app_route.jsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Probably this would be part of the implementation of the registerChannelRoute, it adds the route and wrap the passed component inside a component like this (more or less).